README.md

```diff
@@ -1,519 +1,123 @@
+<p align="center">
+  <br/>
+    <img alt="lighteval library logo" src="./assets/lighteval-doc.svg" width="376" height="59" style="max-width: 100%;">
+  <br/>
+</p>
+
+
+<p align="center">
+    <i>Your go-to toolkit for lightning-fast, flexible LLM evaluation, from Hugging Face's Leaderboard and Evals Team.</i>
+</p>
+
 <div align="center">
 
 [![Tests](https://github.com/huggingface/lighteval/actions/workflows/tests.yaml/badge.svg?branch=main)](https://github.com/huggingface/lighteval/actions/workflows/tests.yaml?query=branch%3Amain)
 [![Quality](https://github.com/huggingface/lighteval/actions/workflows/quality.yaml/badge.svg?branch=main)](https://github.com/huggingface/lighteval/actions/workflows/quality.yaml?query=branch%3Amain)
 [![Python versions](https://img.shields.io/pypi/pyversions/lighteval)](https://www.python.org/downloads/)
 [![License](https://img.shields.io/badge/License-MIT-green.svg)](https://github.com/huggingface/lighteval/blob/main/LICENSE)
-[![Status](https://img.shields.io/pypi/status/lighteval)](https://pypi.org/project/lighteval/)
 [![Version](https://img.shields.io/pypi/v/lighteval)](https://pypi.org/project/lighteval/)
 
 </div>
 
-# LightEval 🌤️
+---
 
-A lightweight framework for LLM evaluation
+**Documentation**: <a href="https://github.com/huggingface/lighteval/wiki" target="_blank">Lighteval's Wiki</a>
 
-## Context
-LightEval is a lightweight LLM evaluation suite that Hugging Face has been using internally with the recently released LLM data processing library [datatrove](https://github.com/huggingface/datatrove) and LLM training library [nanotron](https://github.com/huggingface/nanotron).
+---
 
-We're releasing it with the community in the spirit of building in the open.
+### Unlock the Power of LLM Evaluation with Lighteval 🚀
 
-Note that it is still very much early so don't expect 100% stability ^^'
-In case of problems or questions, feel free to open an issue!
+**Lighteval** is your all-in-one toolkit for evaluating LLMs across multiple
+backends—whether it's
+[transformers](https://github.com/huggingface/transformers),
+[tgi](https://github.com/huggingface/text-generation-inference),
+[vllm](https://github.com/vllm-project/vllm), or
+[nanotron](https://github.com/huggingface/nanotron)—with
+ease. Dive deep into your model’s performance by saving and exploring detailed,
+sample-by-sample results to debug and see how your models stack-up.
 
-## Installation
+Customization at your fingertips: letting you effortlessly create [new
```

Comment:
I would add "or browse all our existing tasks and metrics"

Reply:
changed, wdyt ?

Reply:
nice